kitsune/wiki/jinja2/kb-ux-experiment/enable-and-disable-cookies-website-preferences.html

```diff
@@ -35,9 +35,11 @@ <h2 class="logo">
                   <a href="/en-US/questions/new" class="ask" data-event-category="navigation" data-event-action="main navigation" data-event-label="Get Community Support">Get Community Support</a>
                 </li>
 
-                <li>
-                  <a href="/en-US/users/auth" class="sign-in register" data-event-category="navigation" data-event-action="main navigation" data-event-label="Sign In">Sign In</a>
-                </li>
+                {% if not user.is_authenticated() %}
```

Comment:
It's probably better if we don't include authenticated users in this experiment at all, since a lot about the page can change for an authenticated user, and especially one that has moderation and editing privileges.

Reply:
Sounds good. I will remove this check and I will allow only non-authenticated users from waffle.